### ISSUE TITLE
Issue with logging to Bosh-UI

### ISSUE THREAD

**Issue by Akhilesh-Anb** [cloudfoundry-community/bui-boshrelease#2]
Hi,

I have installed BOSH-UI and i can see the UI in browser.

the default credentials admin/admin is not working. I tried with director credentials and its not working.
I'm facing issue with an error as below

```
2017/05/17 10:34:28 Error unmarshalling info invalid character '<' looking for beginning of value
2017-05-17 10:34:28.36556356 +0000 UTC ./bui: ERROR: login - BOSH Get Password Token invalid character '<' looking for beginning of value
```

Please help me on this. Appreciate your help.

Thanks,
Akhilesh Appana

**Comment by geofffranks** [comment]
It should just pass through your credentials to BOSH. Do you have any special characters (like a '<') in your password?

**Comment by Akhilesh-Anb** [comment]
Hi geoffranks,

Thanks for your response. I'm using bosh-ui as standalone.

I want to know which password it was checking? Is that director credentials?

Thanks,
Akhilesh Appana

**Comment by geofffranks** [comment]
Correct

**Comment by Akhilesh-Anb** [comment]
No. we dont have any speacial character in the password of director.

My config file:-

```
listen_addr: "100.116.16.10:9304"
web_root: ui
skip_ssl_validation: true
bosh_addr: "https://100.116.16.11:25555"
```

I need to add any more details into config file?

**Comment by geofffranks** [comment]
Are you using UAA for authentication on your BOSH?

**Comment by Akhilesh-Anb** [comment]
we are having two types of authentication for bosh login:

1) director credentials
2) token

Thanks,
Akhilesh Appana

**Comment by lnguyen** [comment]
do you have client_id and client_secret in your manifest? https://github.com/cloudfoundry-community/bui-boshrelease/blob/master/jobs/bui/spec#L37-L42

**Comment by Akhilesh-Anb** [comment]
Hi Inguyen,

Thanks for your response.

I'm using BUI, https://github.com/cloudfoundry-community/bui. As no one responded there i raised my question here.

Can i mention the same in my config file ?
```
  bui.uaa.client_id:
    description: "client_id for uaa"
    default: ""
  bui.uaa.client_secret:
    description: "client_secret for uaa"
    default: ""
```

Thanks,
Akhilesh Appana

**Comment by geofffranks** [comment]
We've seen the error you're getting when the `cf` cli is getting an html response while authenticating, instead of json, usually due to bad redirection related to the UAA setup. But if you aren't using UAA auth on your bosh.. maybe something else is returning html instead of json when BUI is trying to log in?

**Comment by lnguyen** [comment]
I've been using `bosh_cli` for `client_id` for uaa. Please note bui is very alpha
